apps/frontend/src/pages/quiz/libs/components/quiz-form/quiz-form.tsx

```diff
@@ -2,17 +2,17 @@ import { QuizQuestion } from "~/libs/components/components.js";
 import { useAppForm } from "~/libs/hooks/hooks.js";
 
 type FormValues = {
-	value: string;
+	answer: "";
 };
 
 const QuizForm: React.FC = () => {
 	const { control } = useAppForm<FormValues>({
-		defaultValues: { value: "" },
+		defaultValues: { answer: "" },
 	});
 
 	return (
 		<form>
-			<QuizQuestion control={control} />
+			<QuizQuestion control={control} name="answer" options={[]} />
```

Comment:
Why empty array for options? Shouldn't be like that

Reply:
I don't use this component, but because I created a type for FormFieldProperties I need to add something to pass Linter tests.